script.js

```diff
@@ -1,22 +1,278 @@
 menuButton.addEventListener('click', () => {
   mobileMenu.style.display = 'flex';
   navMenu.style.display = 'none';
 });
-
 for (let i = 0; i < mobileMenuOptions.length; i += 1) {
   mobileMenuOptions[i].addEventListener('click', () => {
     mobileMenu.style.display = 'none';
     navMenu.style.display = 'block';
   });
 }
-
 navBtnMobileClosed.addEventListener('click', () => {
   mobileMenu.style.display = 'none';
   navMenu.style.display = 'block';
 });
+// Mobile Menu //
+
+// card object data //
+const cardProjectDesk = {
+  name: ['Multi-Post Stories', 'Profesional Art Printing Data', 'Data Dashboard Healthcare', 'Website Portfolio', 'Profesional Art Printing Data', 'Data Dashboard Healthcare', 'Website Portfolio'],
+  description: ["A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.", "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.", "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.", "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.", "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.", "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.", "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard."],
+  featureImage: ['images/works1.png', './images/imgProject1.svg', './images/placeholderCard2.svg', './images/placeholderCard3.svg', './images/placeholderCard1.png', './images/placeholderCard2.svg', './images/placeholderCard3.svg'],
+  linkRef: [
+    'https://developer.mozilla.org/es/docs/Web/CSS',
+    'https://developer.mozilla.org/es/docs/Web/HTML',
+    'https://getbootstrap.com/',
+    'https://www.ruby-lang.org/es/',
+  ],
+  titles: [
+    'CSS definition web page',
+    'HTML definition web page',
+    'Bootstrap definition web page',
+    'Ruby definition web page',
+  ],
+  technologies: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+  linkLive: ['https://ivangonzalez224.github.io/portfolio.io/', 'https://ivangonzalez224.github.io/portfolio.io/', 'https://ivangonzalez224.github.io/portfolio.io/', 'https://ivangonzalez224.github.io/portfolio.io/', 'https://ivangonzalez224.github.io/portfolio.io/', 'https://ivangonzalez224.github.io/portfolio.io/', 'https://ivangonzalez224.github.io/portfolio.io/'],
+  linkSource: ['https://github.com/ivangonzalez224/portfolio.io', 'https://github.com/ivangonzalez224/portfolio.io', 'https://github.com/ivangonzalez224/portfolio.io', 'https://github.com/ivangonzalez224/portfolio.io', 'https://github.com/ivangonzalez224/portfolio.io', 'https://github.com/ivangonzalez224/portfolio.io', 'https://github.com/ivangonzalez224/portfolio.io'],
+};
```

Comment:
- [ ] [Optional] [Highly recommended] It would be better if you arrange the project data as an array that has each project as an object in it like this:

```javascript
const projectsData = [
    {
        name:      ,
        description:     ,
        featureImage:     ,
        technologies:    ,
    },
    {
    ------------
    ------------
    },
    ------------
    ------------
]
```

and then you loop throw this array and render it to the dom.